src/Pos/Ssc/GodTossing/Toss/Pure.hs

```diff
@@ -63,6 +67,21 @@ instance MonadToss PureToss where
     resetShares = PureToss $ gsShares .= mempty
     setEpochOrSlot eos = PureToss $ gsVssCertificates %= VCD.setLastKnownEoS eos
 
+runPureToss
+    :: LoggerName
+    -> MultiRichmenStake
+    -> GtGlobalState
+    -> PureToss a
+    -> (a, GtGlobalState, [LogEvent])
+runPureToss loggerName richmenData gs =
+    convertRes .
+    (\a -> runRWS a richmenData gs) .
+    usingLoggerName loggerName . runPureLog . runNamedPureLogger . getPureToss
```

Comment:
This appears to require `instance (HasLoggerName Identity)`. Should I add this in `log-warper` or in here in this module? Furthermore, what should the identity monad's logger name be?